library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/api/BiomeSelectors.java

```diff
@@ -69,25 +70,23 @@ public static Predicate<BiomeSelectionContext> vanilla() {
 	 * assuming Vanilla's default biome source is used.
 	 */
 	public static Predicate<BiomeSelectionContext> foundInOverworld() {
-		return context -> context.canGenerateIn(DimensionOptions.OVERWORLD);
+		return context -> context.isIn(BiomeTags.IS_OVERWORLD) || context.canGenerateIn(DimensionOptions.OVERWORLD);
 	}
 
 	/**
-	 * Returns a biome selector that will match all biomes that would normally spawn in the Nether,
-	 * assuming Vanilla's default multi noise biome source with the nether preset is used.
+	 * Returns a biome selector that will match all biomes that would normally spawn in the Nether. Datapack Compatible.
```

Comment:
```suggestion
	 * Returns a biome selector that will match all biomes that would normally spawn in the Nether.
	 * This method is compatible with biomes defined on the {@code minecraft:is_nether} tag.
```
If the "Assuming Vanilla's source" line is still true, then it should be readded